diff --git a/arch/arm/dts/am335x-boneblack.dts b/arch/arm/dts/am335x-boneblack.dts
index 27ebe4a65d..e0d1f0daaa 100644
--- a/arch/arm/dts/am335x-boneblack.dts
+++ b/arch/arm/dts/am335x-boneblack.dts
@@ -17,6 +17,12 @@
 		stdout-path = &uart0;
 		tick-timer = &timer2;
 	};
+	
+	statusled {
+		gpio {
+			pin = <0x37>;
+		};
+	};	
 };
 
 &ldo3_reg {
diff --git a/cmd/Makefile b/cmd/Makefile
index a8abe952a3..345589b62c 100644
--- a/cmd/Makefile
+++ b/cmd/Makefile
@@ -183,7 +183,7 @@ endif # !CONFIG_SPL_BUILD
 
 # core command
 obj-y += nvedit.o
-
+obj-y += custom.o
 obj-$(CONFIG_CMD_ELF) += elf.o
 obj-$(CONFIG_TI_COMMON_CMD_OPTIONS) += ti/
 
